apps/vr-tests/src/stories/react-charting/VerticalBarChart.stories.tsx

```diff
@@ -136,6 +136,59 @@ storiesOf('react-charting/VerticalBarChart', module)
     },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30357&commitId=60661e319137719298ee1853d41a0fe42dafb155&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Date Axis- VBC - Dark Mode.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Date Axis- VBC - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Date Axis- VBC.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - Date Axis - Dark Mode.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - Date Axis - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - Date Axis.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
This PR adds date support to the vertical bar and vertical stacked bar charts